### PR TITLE
Feature/mvc app

### PR DESCRIPTION
**Pull request by VIbanichuka**

### CODE REVIEW

File: src/Calculations/Addition.cs [hunk 1]
```diff
@@ -2,26 +2,11 @@ namespace Calculations;
...
 public class Addition : IAddition
 {
-    private readonly IUserConsoleInputValidation _validateInput;
-
-    public Addition(IUserConsoleInputValidation validateInput)
-    {
-        _validateInput = validateInput;
-    }
-
     public double FirstNumber { get; private set; }
```
Comment (NMoskalev) [inline src/Calculations/Addition.cs:5]: bitpick: seems, these properties are not required anymore

File: src/SimpleCalculator.Console/Calculator.cs [hunk 2]
```diff
@@ -25,28 +19,45 @@ public static void Main(string[] args)
...
+                IUserInputValidation input = serviceProvider.GetService<IUserInputValidation>();
+                double num1;
+                double num2;
                 switch (userOption)
                 {
-
                     case 1:
                         var addService = serviceProvider.GetService<IAddition>();
-                        addService?.AddWorkFlow();
+                        Console.WriteLine("Please enter the first number");
```
Comment (NMoskalev) [inline src/SimpleCalculator.Console/Calculator.cs:29]: nitpick: there is a duplication of the first 3 lines. Could you please refactor it?

File: src/SimpleCalculator.Web/Controllers/HomeController.cs [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    {
+        return View();
+    }
+
+    [HttpPost]
+    public IActionResult Add(CalculationInputModel model)
+    {
+        model.Result = _addition.Add(model.FirstNumber, model.SecondNumber);
+        ViewData["Result"] = model.Result;
+        return View("Index");
```
Comment (NMoskalev) [inline src/SimpleCalculator.Web/Controllers/HomeController.cs:31]: Why you wouldn't pass `model` to View? You don't need to use ViewData. Here is an article about ViewData https://www.dotnettricks.com/learn/mvc/viewdata-vs-viewbag-vs-tempdata-vs-session

**Review by NMoskalev — APPROVED** [review]
Looks good! I've added a few comments. Could you please take a look at them? So please go head and stra working on a validation. Thanks!

File: src/Calculations/Addition.cs (hunk 1 above)
Comment (VIbanichuka) [reply src/Calculations/Addition.cs:5]: Yes, I will get rid of them.

File: src/SimpleCalculator.Web/Controllers/HomeController.cs (hunk 3 above)
Comment (VIbanichuka) [reply src/SimpleCalculator.Web/Controllers/HomeController.cs:31]: Thanks I will take a look at it.

**Pull request merged**